packages/common/src/index-client-api.ts

```diff
@@ -0,0 +1,73 @@
+import type { StreamID } from '@ceramicnetwork/streamid'
+import type { StreamState } from './stream.js'
+
+/**
+ * Allowed ordering for an indexing query.
+ */
+export enum Ordering {
```

Comment:
I don't think this should be exposed in the client API.  As we discussed earlier, we want to always use chronological ordering for now, and avoid exposing any ordering options until we've settled on user-facing names for them.

Reply:
Again, let me remind you about tradeoffs here. It is easier to remove the option than to add it at a later time.